pkg/networks/commands.go

```diff
@@ -23,22 +28,60 @@ func (config *NetworksConfig) Check(name string) error {
 	return fmt.Errorf("network %q is not defined", name)
 }
 
+// IsDaemonInstalled checks whether the daemon is installed.
+func (config *NetworksConfig) IsDaemonInstalled(daemon string) (bool, error) {
+	var p string
+	switch daemon {
+	case VDESwitch:
+		p = config.Paths.VDESwitch
+	case VDEVMNet:
+		p = config.Paths.VDEVMNet
+	case SocketVMNet:
+		p = config.Paths.SocketVMNet
+	default:
+		return false, fmt.Errorf("unknown daemon type %q", daemon)
+	}
+	if p == "" {
+		return false, nil
+	}
+	if _, err := exec.LookPath(p); err != nil {
+		if errors.Is(err, fs.ErrNotExist) {
+			return false, nil
+		}
+		return false, err
+	}
+	return true, nil
+}
+
+// Sock returns a socket_vmnet socket.
+func (config *NetworksConfig) Sock(name string) string {
+	return fmt.Sprintf("%s/socket_vmnet.%s", config.Paths.VarRun, name)
```

Comment:
Nit: all the other functions like `VDESock` and `PIDFile` use `filepath.Join()` for the same functionality:

```suggestion
	return filepath.Join(config.Paths.VarRun, fmt.Sprintf("socket_vmnet.%s", name))
```

Either match existing style or change the other locations to match the new style.

Reply:
Fixed